docs/DeveloperGuide.md

```diff
@@ -1038,10 +1038,10 @@ Steps 2 and 3 are explained in greater details in the following sequence diagram
 day in the schedule stated by the user. It will fist call the `String#split(" ")` method 
 to separate out the `userArgument` given by the user. Upon, splitting of the whitespaces in `userArgument`, 
 it will then check if the `userArgument` is valid. If it is invalid, an 
-`Exception` would be thrown to the user and following the termination of the process (step 2.3).
+`ArrayOutOfBoundException` would be thrown to the user and following the termination of the process (step 2.3).
```

Comment:
I think the exception's name is `ArrayIndexOutOfBoundsException`

Reply:
thanks for pointing out this mistake